db/migrate/20220707152130_create_cars.rb

```diff
@@ -0,0 +1,14 @@
+class CreateCars < ActiveRecord::Migration[7.0]
+  def change
+    create_table :cars do |t|
+      t.string :name
+      t.text :description
+      t.string :image_url
+      t.string :city
+      t.integer :user_id
+      t.integer :price
```

Comment:
- [ ] I think to create a car we just need the `name`, `model` or `description` and the `price`. Also, I think the `city` is for a reservation because when we reserve a car I think we need to specify the city.
- [ ] For the `user_id` Jerry has already left a comment.